docs/blogs/EnhancedDevPreviewGatewayAPI/IntroEnhancedDevPreview.md

```diff
@@ -0,0 +1,45 @@
+# Gateway API with OpenShift Networking - Developer Preview Update
+
+Authors: Candace Holman and Grant Spence
+
+![Gateway API Dev Preview Update](images/gateway-api-dev-preview-update.png)
+
+As a follow-on to the Openshift 4.12 Developer Preview of Gateway API, OpenShift has released an enhanced Developer
+Preview for Gateway API support in OCP 4.13.5. If you haven't read our first blog post, [Introducing Gateway API with OpenShift Networking - Developer Preview](https://cloud.redhat.com/blog/introducing-gateway-api-with-openshift-networking-developer-preview),
+which contains an overview of the Gateway API components, please take a minute to review it.
+
+## New Features in Developer Preview
+
+With this Developer Preview update, developers can now install and configure the capability to use Gateway API with the
+Ingress Operator, seamlessly installing all the necessary Gateway API components without any external downloads. By
+delegating the responsibility of installation and configuration to the Ingress Operator, developers can now dedicate
+their focus to exploring and leveraging the powerful API that Gateway API has to offers.
+
+The Ingress Operator enables the use of Gateway API by:
+* Installing version 0.5.1 Gateway API custom resource definitions (CRDs)
+* Installing and configuring Istio as the Gateway API implementation via [OpenShift Service Mesh (OSSM)](https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/ossm-about.html)
+* Enabling creation of Load Balancers via Istio’s [automated deployments](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment)
+* Automatically creating DNS Records for Gateways
+
+These updates provide OpenShift developers with the opportunity to explore and utilize Gateway API using our existing
+range of product offerings. By integrating the Ingress Operator with OpenShift Service Mesh, we are able to offer this
+Developer Preview of Gateway API with our existing robust and security-focused Istio solution.
+
+### Behind a Feature Gate
+
+The features mentioned in this Developer Preview update are accessible through [OpenShift's feature gate mechanism](https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-enabling-features.html).
+By implementing Gateway API support under a feature gate, it is not activated by default, requiring users to explicitly
+opt in to use it. The feature gate is a part of the “customNoUpgrade” featureSet, which means the cluster will no longer
```

Comment:
It's a minor detail, but [the "CustomNoUpgrade" feature set itself](https://github.com/openshift/api/blob/afcbe27aec7ce434d68c98796208a534fdbbf0ea/config/v1/types_feature.go#L161-L164) is empty; enabling this feature set then allows the cluster-admin [to specify individual features in the cluster config](https://github.com/openshift/api/blob/afcbe27aec7ce434d68c98796208a534fdbbf0ea/config/v1/types_feature.go#L64-L69).
```suggestion
opt in to use it. The feature gate can be enabled by configuring the “CustomNoUpgrade” feature set, which means the cluster will no longer
```

Reply:
Done.